### PR TITLE
NOOS-517/cross-platform-docker-images

### PR DESCRIPTION
**Pull request by zyassine**

### CODE REVIEW

**Comment by zyassine** [comment]
@pieeri I didn't update Docker version because I get this error in circleci `Job was rejected because this version of Docker is not supported on this resource class. Try removing the version of Docker in your .circleci/config.yml and re-running`
`buildx` is supported in version 20.10.14

File: src/commands/docker_buildx_image.yml [hunk 1]
```diff
@@ -0,0 +1,203 @@
+---
+
+description: Build, tag and push Docker image to remote registry [AWS ECR].
```
Comment (pieeri) [inline src/commands/docker_buildx_image.yml:3]: 🐼  Maybe add "for cross platform" in the description to differentiate command from the standard `buil` command?

**Review by pieeri — CHANGES_REQUESTED** [review]
❓ 

What about using the `default` flag in the Docker remote version to keep up-to-date with the latest Docker engine, while being compatible with the underlying hardware in CircleCI?

https://circleci.com/docs/building-docker-images/#docker-version

**Review by pieeri — APPROVED** [review]
🚢 !

**Pull request merged**